packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -167,6 +168,16 @@ class RefreshIndicator extends StatefulWidget {
   /// [Future] must complete when the refresh operation is finished.
   final RefreshCallback onRefresh;
 
+  /// This bool can be changed to show and hide the refresh indicator reactively.
+  /// Defaults to false and if set to true before [State.initState] will show
+  /// the refresh indicator as soon as it is built.
+  ///
+  /// Note that reactively showing the refresh indicator will not call [onRefresh]
+  /// or automatically hide the indicator after the returned [Future] completes.
+  /// If `isRefreshing` has been set to `true`, it must be set to `false` through
+  /// your own logic to hide the indicator and allow refreshing by dragging again.
+  final bool isRefreshing;
```

Comment:
Would this make sense to be named `initiallyRefreshing` or `manualRefreshing`? The reason I ask is that if it is false, but the user initiates a refresh from a drag action, `isRefreshing` will still be false but it will be refreshing.

Reply:
I agree with this, it feels like there should be a better name but I couldn't come up with one. `manuallyRefreshing` might also seem like it should refer to when the user is manually refreshing by dragging the scrollable? And `initiallyRefreshing` doesn't really reflect that it can be used arbitrarily.

Changed it to `manuallyRefreshing` since it does make more sense in a way.